view/frontend/web/template/payment/offsite-installment-form.html

```diff
@@ -0,0 +1,184 @@
+<div class="payment-method" data-bind="css: {'_active': (getCode() == isChecked())}">
+    <div class="payment-method-title field choice">
+        <input type="radio" name="payment[method]" class="radio" data-bind="attr: {
+               'id': getCode()
+               },
+               value: getCode(),
+               checked: isChecked,
+               click: selectPaymentMethod,
+               visible: isRadioButtonVisible(),
+               enable: isActive() && !orderValueTooLow()" />
+        <label data-bind="attr: {'for': getCode()}" class="label">
+            <span data-bind="text: getTitle()"></span>
+        </label>
+        <div data-bind="visible: !isActive()" class="page messages">
+            <div role="alert" class="messages">
+                <div class="message-warning warning message">
+                    <div data-bind="i18n: 'Option available only for orders in THB'"></div>
+                </div>
+            </div>
+        </div>
+        <div data-bind="visible: orderValueTooLow()" class="page messages">
+            <div role="alert" class="messages">
+                <div class="message-warning warning message">
+                    <div data-bind="i18n: 'Minimum order value is 5000 THB'"></div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="payment-method-content">
+        <!-- ko foreach: getRegion('messages') -->
+        <!-- ko template: getTemplate() -->
+        <!-- /ko -->
+        <!--/ko-->
+        <div class="payment-method-billing-address">
+            <!-- ko foreach: $parent.getRegion(getBillingAddressFormName()) -->
+            <!-- ko template: getTemplate() -->
+            <!-- /ko -->
+            <!--/ko-->
+        </div>
+        <div class="omise-banking-fee-warn" data-bind="i18n: 'Choose your installment terms'"></div>
+        <form class="form" data-bind="attr: {
+              id: getCode() + 'Form',
+              }">
+
+            <fieldset data-bind="attr: {class: 'fieldset payment items ' + getCode(), id: 'payment_form_' + getCode()}">
+                <ul class="form-list">
+                    <!-- KTC -->
+                    <li class="items">
+                        <input type="radio" name="offsite" value="installment_ktc" data-bind="
```

Comment:
Why are these harcoded? i.e. each individual instalment type? They should be coming from the Capabilities API response (dynamically generated)

Reply:
Good idea... this will be subject for future PR, for now we have only limited number specified providers.

Reply:
@jacstn Just wonder, is there any reason it won't be in this PR? Cuz this seems major one to me.
i.e. I have `installment_bay` in my Capabilities API but it doesn't show in the UI because of the hard-code.
Also some merchants might have different setting on which banks they can accept instalment?

Reply:
@guzzilar it is knockout.js.. and I am not sure how to do it yet. 
in the same time I spoke to sales team and to product team, it is fine to hardcode it now, according to Visit we have no situation when we would limit number of installment options to our customers.

Releasing it is main priority

Reply:
@jacstn Then can you make sure that the list of instalment will be just only 4? Because the `installment_bay` is still missing here..

Reply:
yes.. it is missing.. and needs to be added..